Sources/Controllers/MyPlaces/TracksViewController.swift

```diff
@@ -405,8 +459,40 @@ class TracksViewController: OACompoundViewController, UITableViewDelegate, UITab
     }
     
     @objc private func onRefresh() {
-        updateAllFoldersVCData()
-        tableView.refreshControl?.endRefreshing()
+        gpxDB.load()
```

Comment:
`load()` should not be called here (intended to call once only - at app startup)